routes/index.js

```diff
@@ -6,13 +6,16 @@ router.get('/', function(req, res, next) {
   res.render('index', {
     title: 'chubbo-chat',
     //pass src for page bkground only needed on landing
-    background: '/javascripts/floatingDots.js'
+    script: '/javascripts/floatingDots.js'
```

Comment:
Why did you add this as a variable instead of hardcoding it in the .pug file? What value does this method add?